docs/source/usage/plugins/radiation.rst

```diff
@@ -287,6 +287,10 @@ Command line option                       Description
 ``--<species>_radiation.folderRadPerGPU`` Name of the folder, where the GPU specific spectra are stored.
                                           Default: ``radPerGPU``
 ``--<species>_radiation.compression``     If set, the hdf5 output is compressed.
+``--<species>_radiation.numTmpResults``   Number of temporary results per device.
+                                          This option is used to increase the utilization of the device by producing more independent work.
+                                          Increase results in more independent parallel work and a larger memory footprint.
+                                          Default: ``2``
```

Comment:
As a matter of consistency I think the default should be 1.

Until we are reasonably sure that:

- This approach does not introduce some weird error.
- It's not detrimental to performance on e.g., K20 and K80 systems.
(admittedly test for both are ongoing).

Reply:
IMO K20 and k80 are not interesting anymore. The hardware is nearly a decade old. If you run on such systems you should adjust it by hand if required. 

`This approach does not introduce some weird error.` 

This is true for each code change. Therefore you are running currently some tests but if this PR is not introducing an general error during the reduce of the tmp arrays it is not critical.

Reply:
I agree with @Anton-Le if the physical correctness has not yet been confirmed for the k20 and k80 platform. If this still produces the same result, I would be fine with a default of two.

Reply:
The changes are independent of the GPU architecture, so the tests can take place on any GPU hardware.
@Anton-Le could you please confirm the correctness of the results based on your tests.

Reply:
After the recent re-run of the reference simulation with the version from 4.08.2020 the maximum relative error in the  spectrum appears to be ~ 0.02 (for forward direction) and commonly ranges at ~ 0.004.
Hence I would consider this as correct.

Reply:
@Anton-Le Thanks for testing the code.